frontends/mit-open/src/page-components/Header/Header.tsx

```diff
@@ -7,48 +7,137 @@ import {
   NavDrawer,
   Toolbar,
   ClickAwayListener,
+  ActionButtonLink,
 } from "ol-components"
+import { RiSearch2Line } from "@remixicon/react"
 import { MITLogoLink, useToggle } from "ol-utilities"
 import UserMenu from "./UserMenu"
 import { MenuButton } from "./MenuButton"
 import {
   DEPARTMENTS,
   RESOURCE_DRAWER_QUERY_PARAM,
+  SEARCH,
   querifiedSearchUrl,
 } from "@/common/urls"
 import { useSearchParams } from "react-router-dom"
+import { useUserMe } from "api/hooks/user"
 
-const Bar = styled(AppBar)`
-  height: 56px;
-  padding: 0 8px;
-  z-index: ${({ theme }) => theme.zIndex.drawer + 1};
-  background-color: ${({ theme }) => theme.custom.colors.white};
-  color: ${({ theme }) => theme.custom.colors.black};
-  display: flex;
-  flex-direction: column;
-  box-shadow: 0 2px 10px rgba(120 169 197 / 15%);
-`
+const Bar = styled(AppBar)(({ theme }) => ({
+  height: "80px",
+  padding: "0 8px",
+  zIndex: theme.zIndex.drawer + 1,
+  backgroundColor: theme.custom.colors.white,
+  color: theme.custom.colors.darkGray1,
+  display: "flex",
+  flexDirection: "column",
+  boxShadow: "0 2px 10px rgba(120 169 197 / 15%)",
+  [theme.breakpoints.down("sm")]: {
+    height: "60px",
+    padding: "0",
+  },
+}))
+
+const FlexContainer = styled.div({
+  display: "flex",
+  alignItems: "center",
+})
+
+const DesktopOnly = styled(FlexContainer)(({ theme }) => ({
+  [theme.breakpoints.up("sm")]: {
+    display: "flex",
+  },
+  [theme.breakpoints.down("sm")]: {
+    display: "none",
+  },
+}))
+
+const MobileOnly = styled(FlexContainer)(({ theme }) => ({
+  [theme.breakpoints.down("sm")]: {
+    display: "flex",
+  },
+  [theme.breakpoints.up("sm")]: {
+    display: "none",
+  },
+}))
 
 const StyledToolbar = styled(Toolbar)({
   flex: 1,
 })
 
-const LogoLink = styled(MITLogoLink)({
-  width: 45,
-  height: "auto",
+const LogoLink = styled(MITLogoLink)(({ theme }) => ({
```

Comment:
The logo looks a little off the vertical center in mobile view:

<img width="239" alt="image" src="https://github.com/mitodl/mit-open/assets/939376/c1d1bf50-68e4-4f3a-bf93-35bc4a3943aa">

Reply:
(cc @steven-hatch ) Yea, this is likely due to the placeholder nature of the logo image. If you look at it in inspector, the content is not centered within the viewbox of the SVG: 

<img width="100" alt="image" src="https://github.com/mitodl/mit-open/assets/12089658/812bf8bb-d9f7-4f6b-8e61-8cf497399b99">